app/views/content_items/detailed_guide.html+universal_navigation.erb

```diff
@@ -0,0 +1,41 @@
+<%= content_for :simple_header, true %>
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', @content_item.title_and_context %>
+    <%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
+  </div>
+  <div class="column-third">
+    <%= render 'components/translation-nav', translations: @content_item.available_translations %>
+    <% if @content_item.image.present? %>
+      <%= image_tag @content_item.image, class: "logo-image" %>
+    <% end %>
+  </div>
+</div>
+
+<%= render 'components/content-metadata', {
+      published: @content_item.published,
+      last_updated: @content_item.updated,
+      link_to_history: !!@content_item.updated,
+      from: @content_item.organisations_ordered_by_importance
+    } %>
+
+<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
+<%= render 'shared/history_notice', content_item: @content_item %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <div class="responsive-bottom-margin">
+      <%= render 'components/contents-list', contents: @content_item.contents %>
```

Comment:
In some guides, the responsive-bottom-margin div is rendering with nothing in it, making the gap between metadata and content look quite large. I think this is happening when there's no contents list, but the wrapping div still renders?

https://government-frontend-pr-535.herokuapp.com/guidance/introduction-of-statutory-participation-in-trialling-of-national-curriculum-tests-from-2016

<img width="1002" alt="screen shot 2017-11-14 at 13 23 04" src="https://user-images.githubusercontent.com/29889908/32782038-fa10942e-c93e-11e7-9759-acdb782397f2.png">